react-carousel/my-app/src/Dots.tsx

```diff
@@ -0,0 +1,22 @@
+import { GoDot } from 'react-icons/go';
+import { GoDotFill } from 'react-icons/go';
+type Images = {
```

Comment:
Import this from `Carousel` instead of defining it again here. You don't want to repeat yourself!